src/macaron/slsa_analyzer/checks/detect_malicious_metadata_check.py

```diff
@@ -141,16 +141,15 @@ class MaliciousMetadataFacts(CheckFacts):
 
 
 class DetectMaliciousMetadataCheck(BaseCheck):
-    """This check analyzes the metadata of the pypi package based on seven heuristics."""
+    """This check analyzes the metadata of a package based on reports malicious behavior."""
```

Comment:
> based on reports malicious behavior
Not sure what is supposed to be conveyed here.

Reply:
See commit [728d753](https://github.com/oracle/macaron/pull/816/commits/728d753678ee948b303d9b2fb9c672f245ed7ee0)